linode/manager
# Organize ts configurations alphabetically

## Description 📝

- Organize TS configurations alphabetically
- disable allowUnreachableCode

**What does this PR do?**
Organizes the TS configurations alphabetically, This makes life easy to locate any specific config and improves readability.
Disable allowUnreachableCode, which helps to identify any unreachable code at compile time.
Example: 
```
function doSomething(): never {
   while (true) {
   }
}

doSomething()
console.log('log something...') // throws error saying "unreachable code detected"
```
## How to test 🧪
Run below to see any compilation errors and run all unit tests.

- yarn up 
- yarn test

**How do I run relevant unit or e2e tests?**
`yarn cy:run`
